packages/beacon-node/src/network/network.ts

```diff
@@ -105,7 +103,6 @@ export class Network implements INetwork {
     this.chain.emitter.on(routes.events.EventType.head, this.onHead);
     this.chain.emitter.on(routes.events.EventType.lightClientFinalityUpdate, this.onLightClientFinalityUpdate);
     this.chain.emitter.on(routes.events.EventType.lightClientOptimisticUpdate, this.onLightClientOptimisticUpdate);
-    modules.signal.addEventListener("abort", this.close.bind(this), {once: true});
```

Comment:
This is a terrible idea, debugging tests I discovered that close() is being called twice:
- first time through AbortController, not awaited, uncontrolled
- second time via close() but since it's second time it has no effect

This broke the expected close flow from the caller, causing issues in subsequent tests